examples/deployments/k8s_deploy/README.md

```diff
@@ -25,8 +25,8 @@ Before running this example, ensure you have:
 
 1. **Install dependencies**:
    ```bash
-   pip install agentscope-runtime==0.1.5b1
-   pip install "agentscope-runtime[deployment]==0.1.5b1"
+   pip install agentscope-runtime>=1.0.0
```

Comment:
Since we got `pip install "agentscope-runtime[deployment]>=1.0.0"`, this line can be removed.